backend/dal/Migrations/v01.12.02/Up/PreUp/01-AdministrativeAreas.sql

```diff
@@ -0,0 +1,4 @@
+PRINT 'Removing duplicate salmon arm'
+
+DELETE FROM dbo.[AdministrativeAreas]
+WHERE [Id] = 248 -- Salmon Arm Tsey Key duplicate
```

Comment:
You should probably ensure any possible duplicates are dealt with.  There is no guarantee the other environment DBs have the same data as you do locally.
Additionally, make sure nothing references these as a foreign key.  I don't think they do, I think we use it only as a list.

Reply:
Hey! Just need to address this one still but knocked off the other requests. To clarify - do you mean add to this PreUp script to check for duplicates in the system?

Reply:
It can be in this script.  It doesn't need to be in the preup.

For example if you don't ensure there are no other duplicates.  Then the migration will fail.  You need to make sure there are no other possible duplicates.  I'd lean towards not hardcoding the delete for an ID.  As a practice the IDs could be different in each environment.